recipes/anaquin/build.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set -eu -o pipefail
+
+# download and copy required dependencies
+# klib
+wget https://github.com/attractivechaos/klib/archive/spawn-final.tar.gz
+tar xzf spawn-final.tar.gz
+cp -r $SRC_DIR/klib-spawn-final $PREFIX/lib/klib/
+
+# vcflib
+wget https://github.com/student-t/vcflib-forAnaquin/archive/v1.0.0-rc1.tar.gz
+tar xzf v1.0.0-rc1.tar.gz
```

Comment:
vcflib is also available in bioconda:
https://github.com/bioconda/bioconda-recipes/tree/master/recipes/vcflib

but it only provides the binaries (which is sensible):
https://github.com/bioconda/bioconda-recipes/blob/master/recipes/vcflib/build.sh

Also, the author of anaquin modified the source code to work with it, as explained here:
https://s3.amazonaws.com/sequins/software/Anaquin_CompileFromSource_v1.pdf

so not sure what to do here?